Sources/Jetworking/Client/Interceptor/Request/HeaderFieldsRequestInterceptor.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 /// Implementation of a request interceptor which adds header fields to the request.
-public class HeaderFieldsRequestInterceptor: RequestInterceptor {
+public class HeaderFieldsRequestInterceptor: Interceptor {
```

Comment:
Shouldn't we name it `HeaderFieldsInterceptor` instead of `HeaderFieldsRequestInterceptor`?